packages/deeplinks/observability/deep_links.ts

```diff
@@ -27,6 +27,8 @@ export type AppId =
   | ApmApp
   | MetricsApp;
 
+export type DiscoverLogExplorerId = 'discover:log-explorer';
```

Comment:
Sorry for being so picky :innocent: would it make sense to still interpolate `DISCOVER_APP_ID` here?